packages/grafana-ui/src/components/FormField/FormField.tsx

```diff
@@ -30,6 +31,7 @@ export const FormField: FunctionComponent<Props> = ({
   inputWidth,
   inputEl,
   className,
+  grow,
```

Comment:
can you explain the reason for adding the grow property?

Reply:
Since the FormField property is reusable, the same applies to the style.

Adding the `grow` prop to this component makes it easier to activate the `flex-grow` CSS property depending on the context in which this component is used.

Reply:
Oh...just noticed the `grow` prop isn't used within and outside the FormField component.
Figured it's best to remove this to avoid unnecessary props.